client/src/pages/inventory/InventoryCreate.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Container,
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Alert,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Dialog
+} from '@mui/material';
+
+import { Dropzone, FileItem } from "@dropzone-ui/react";
+import PageHeader from '../../components/page-header/PageHeader';
+import Spacer from '../../components/spacer/Spacer';
+import VaraintCreateDialog from './VariantCreate';
+import { useNavigate } from 'react-router-dom';
+
+const variantList = [
+  { name: "Rainbow Kirbyzz", price: "$420.69", date: "1999-07-29", barcode: "66666"},
+  { name: "Chad Kirby", price: "$420.69", date: "1999-07-29", barcode: "66666"},
+  { name: "Snacccc Kirby", price: "$420.69", date: "1999-07-29", barcode: "66666"},
+];
+
+const VariantList = () => (
+  variantList.map(({ name, price, date, barcode }) => (
+  <TableRow>
+      <TableCell>{name}</TableCell>
+      <TableCell>{price}</TableCell>
+      <TableCell>{date}</TableCell>
+      <TableCell>{barcode}</TableCell>
+  </TableRow>
+)
+))
+export default function InventoryCreatePage() {
+
+  const navigate = useNavigate();
+  const [variantDialogOpen, setVariantDialogOpen] = useState(false);
+  return (
+    <Container>
```

Comment:
This page is nearly identical to the edit page, can we merge them into a single page?